Modules/Registration/Common/include/itkPointsLocator.h

```diff
@@ -95,17 +95,13 @@ class ITK_TEMPLATE_EXPORT PointsLocator : public Object
   PointIdentifier
   FindClosestPoint(const PointType & query) const;
 
-  /** Find the k-nearest neighbors.  Returns the point ids. */
-  void
-  Search(const PointType &, unsigned int, NeighborsIdentifierType &) const;
-
   /** Find the closest N points.  Returns the point ids. */
   void
   FindClosestNPoints(const PointType &, unsigned int, NeighborsIdentifierType &) const;
 
-  /** Find all the points within a specified radius.  Returns the point ids. */
+  /** Find the closest N points.  Returns the point ids and save the distance */
```

Comment:
"Return the point ids and save the distances."
or
"Returns the point ids and saves the distances."

Although I think the latter is discouraged.